BootloaderCorePkg/Stage1A/Stage1A.c

```diff
@@ -361,13 +362,28 @@ SecStartup2 (
 
   // Extra initialization
   if (FlashMap != NULL) {
-    SetCurrentBootPartition ((FlashMap->Attributes & FLASH_MAP_ATTRIBUTES_BACKUP_REGION) ? 1 : 0);
+    if (PcdGetBool (PcdSblResiliencyEnabled)) {
```

Comment:
Move back to platform-specific code... can still abstract into library within individual platform.

Run into issue with a lot of dummy code if we make it fully common.

Reply:
Actually don't move this yet.